src/test/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtilsUnitTest.java

```diff
@@ -144,6 +144,32 @@ public void testQueryWithNullLabel() {
         checkQueryResults(result, getAllExpectedNamesAndAges(), query);
     }
 
+    @Test(groups = {"cloud"})
+    public void testQueryWithNullWorkflow_Name() { // THIS IS THE TEST IM WRITING
+        final Map<String, String> expectedNamesAndAges = getAllExpectedNamesAndAges();
```

Comment:
unused variable and no side effects in the method invocation so I think this can be deleted